src/Builder.php

```diff
@@ -422,6 +426,13 @@ private function getFilePath(Request $request): array
         return [$info['dirname'], $info['basename']];
     }
 
+    private function isHtml(Request $request, Response $response): bool
+    {
+        $contentType = explode(';', $response->headers->get('Content-Type'))[0];
```

Comment:
I guess we could always consider it as html if for any reason there is no content-type header at all.

```suggestion
        $contentType = explode(';', $response->headers->get('Content-Type', 'text/html'))[0];
```